Dockerfiles/cluster-agent/dist/templates/header.tmpl

```diff
@@ -28,13 +28,17 @@
     {{$name}}: {{$value}}
     {{- end }}
   {{- end }}
-      hostname provider: {{.hostnameStats.provider}}
+    hostname provider: {{.hostnameStats.provider}}
+  {{- if gt (len .hostnameStats.errors) 0 }}
+    unused hostname providers:
+  {{- end }}
   {{- range $name, $value := .hostnameStats.errors -}}
-    {{- if ne $name "all" }}
-    error: {{$name}}: {{$value}}
-    {{- else }}
-    error: {{$value}}
-    {{- end}}
+      {{- if ne $name "all" }}
+      {{$name}}: {{$value}}
```

Comment:
nit - indent

Reply:
this weird indentation is necessary for the resulting status page to have the correct indentation (problem of code indentation vs. indentation of what's actually being displayed)